sdk/Lusid.Sdk.Tests/tutorials/Properties/TransactionProperties.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Client;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Properties
+{
+    [TestFixture]
+    public class TransactionProperties : TutorialBase
+    {
+        private IList<string> _instrumentIds;
+        
+        [OneTimeSetUp]
+        public void OnetimeSetUp()
+        {
+            var instrumentLoader = new InstrumentLoader(_apiFactory);
+            _instrumentIds = instrumentLoader.LoadInstruments();
+        }
+
+        internal string _portfolioCode;
+        [SetUp]
+        public void SetUp()
+        {
+            var portfolioRequest = TestDataUtilities.BuildTransactionPortfolioRequest();
+            var portfolio = _transactionPortfoliosApi.CreatePortfolio(TestDataUtilities.TutorialScope, portfolioRequest);
+            _portfolioCode = portfolioRequest.Code;
+            Assert.That(portfolio?.Id.Code, Is.EqualTo(_portfolioCode));
+        }
+
+        [LusidFeature("F13-4")]
+        [Test]
+        public void Test_Transaction_Property()
```

Comment:
these should be considered examples rather than tests so I would rename to be more descriptive, something like `Add_Transactions_To_Portfolio_With_Property`

Reply:
Done. Originally named to match Python.